src/marten/cli/manage/command/collect_assets.cr

```diff
@@ -6,10 +8,21 @@ module Marten
           command_name :collectassets
           help "Collect all the assets and copy them in a unique storage."
 
+          @fingerprint : Bool = false
+          @fingerprint_mapping = Hash(String, String).new
           @no_input : Bool = false
+          @manifest_path : String = File.join Marten.apps.main.class._marten_app_location, "manifest.json"
 
           def setup
+            on_option("fingerprint", "Add a fingerprint to the collected assets") { @fingerprint = true }
             on_option("no-input", "Do not show prompts to the user") { @no_input = true }
+            on_option_with_arg(
+              "manifest-path",
+              arg: "Filepath",
+              description: "Specify where the manifest file should be stored."
```

Comment:
I think we can also add a mention of the default path in the option description! 😊